### DISCUSSION TITLE
Can we enable audi heartbeat sound on B9?

### DISCUSSION THREAD

**Discussion by Jackie264** [Mr-MIBonk/M.I.B._More-Incredible-Bash#145] [General]
Thanks

**Comment by harman-f** [comment]
M.I.B. -> Multimedia System -> AUDI -> Coming & Leaving Sounds

**Reply by Jackie264** [reply]
Thanks.
I've done this.
Both "Vehicle Readiness Sound - Audi only" and "Vehicle Leaving Sound - Audi only" are checked, then finished "Copy Sound Files", and reboot the unit.
Everything's ok, but only the readiness sound suceed. The leaving sound dosen't work.
What else can I do?
